cmd/ops_agent_windows/run_windows.go

```diff
@@ -207,6 +222,12 @@ func run(name string) error {
 	}
 	defer elog.Close()
 
+	// Redirect stdout to the event log to capture internal messages
+	log.SetOutput(&eventLogWriter{
+		EventID:  StdoutEventID,
+		EventLog: elog,
```

Comment:
In this setup, what is the os.stdout being binded to io.Writer of log.SetOutput? 🤔 

Is there test added for verifying the warning message that was previously lost now is tracked in eventlog?

Reply:
This test verifies the warning message which was previously lost: https://github.com/GoogleCloudPlatform/ops-agent/pull/1086/files#diff-4c9b98fddf589bb837ae19fbe885c647cdb7d97f048fc2e742e84c0d80554152R1664